### PR TITLE
Fix Injector Link

### PR DESCRIPTION
**Pull request by N0tExisting**
Fixed Injector Link, so idiots wont complain about it

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ If you want to contribute to this project, feel free to fork this repository and
 **As of 09/01/20, this project is under no active development by the horion team anymore. We will still review PRs.**
 
 [![Discord](https://img.shields.io/discord/732833913705201736?label=chat&logo=discord&logoColor=white&style=for-the-badge)](https://discord.gg/WPe3WeNtgv)
-[![Download the injector](https://img.shields.io/badge/download-injector-brightgreen?style=for-the-badge "Download the injector")](https://horionbeta.club/injector "Download the injector")
+[![Download the injector](https://img.shields.io/badge/download-injector-brightgreen?style=for-the-badge "Download the injector")](https://github.com/horionclient/Horion/raw/master/Injector/HorionRelease.zip "Download the injector") <!--https://horionbeta.club/injector-->
```
Comment (N0tExisting) [inline README.md:11]: This link can only work if this is merged.

**Review by Mixu78 — APPROVED** [review]
yes

**Comment by xAffan** [comment]
PLS CHECK IF THE ZIP HAS BEEN TAMPERED WITH OR NOT **BEFORE MERGING**

File: README.md (hunk 1 above)
Comment (LeonetKenner) [reply README.md:11]: Oof this link doesnt work at all

Comment (N0tExisting) [reply README.md:11]: > Oof this link doesnt work at all

Did you read my comment?

Comment (LeonetKenner) [reply README.md:11]: when will they merge lmao

**Comment by Mixu78** [comment]
wtf

**Comment by xAffan** [comment]
L

**Pull request closed without merging**